e2e-tests/cypress/integration/before-you-start-appellant-selects-the-site.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Appellant selects the site
```

Comment:
Nothing to do with this PR, but Scenario-4 and Error message -2 does not sync up. i.e. Error message 2 will never be shown

Reply:
Corrected the test implementation